spec/javascripts/support/jasmine_helper.rb

```diff
@@ -78,4 +88,7 @@ def call(env)
 
   # serve weback-compiled packs from public/packs/ on /packs/
   config.add_rack_path('/packs', -> { WebpackPack.new })
+
+  cli_options = Jasmine::Runners::ChromeHeadless.new(nil, nil, config).cli_options_string
+  puts "Jasmine using #{chrome_version}: \"#{chrome_binary}\" #{cli_options}"
```

Comment:
This printed the following for me locally, which is useful for diagnostics and debugging:

```
Jasmine using Google Chrome 110.0.5481.100: "/Applications/Google Chrome.app/Contents/MacOS/Google Chrome" --headless=new --disable-gpu --remote-debugging-port=9222
```